app/bundles/FormBundle/Tests/EventListener/ReportSubscriberTest.php

```diff
@@ -58,13 +74,25 @@ public function setUp(): void
 
         $this->companyReportData    = $this->createMock(CompanyReportData::class);
         $this->submissionRepository = $this->createMock(SubmissionRepository::class);
+        $this->formModel            = $this->createMock(FormModel::class);
         $this->formRepository       = $this->createMock(FormRepository::class);
+
+        $createReportHelper         = \Closure::bind(
+            function () {
+                return new ReportHelper(new EventDispatcher());
+            },
+            $this,
+            ReportHelper::class
+        );
+        $this->reportHelper         = $createReportHelper();
```

Comment:
What is the goal of this?

Reply:
I was going to use it instead of the mock object. It looks like I can do without it.